commercetools-models/src/main/java/io/sphere/sdk/products/queries/ProductByKeyHeadImpl.java

```diff
@@ -0,0 +1,39 @@
+package io.sphere.sdk.products.queries;
+
+import io.sphere.sdk.http.NameValuePair;
+import io.sphere.sdk.products.Product;
+import io.sphere.sdk.products.expansion.ProductExpansionModel;
+import io.sphere.sdk.products.search.PriceSelection;
+import io.sphere.sdk.queries.MetaModelGetDslImpl;
+import io.sphere.sdk.queries.MetaModelHeadDsl;
+import io.sphere.sdk.queries.MetaModelHeadDslBuilder;
+import io.sphere.sdk.queries.MetaModelHeadDslImpl;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+import static io.sphere.sdk.client.SphereRequestUtils.urlEncode;
+import static io.sphere.sdk.products.search.PriceSelectionQueryParameters.extractPriceSelectionFromHttpQueryParameters;
+import static io.sphere.sdk.products.search.PriceSelectionQueryParameters.getQueryParametersWithPriceSelection;
+
+final class ProductByKeyHeadImpl extends MetaModelHeadDslImpl<Product, Product, ProductByKeyHead, ProductExpansionModel<Product>> implements ProductByKeyHead {
```

Comment:
The Head methods don't have expansion, price selection etc

Reply:
thanks, I followed the wrong PR about the price selection